spec/gogo/cmd/bnsd/x/blueaccount/codec.proto

```diff
@@ -0,0 +1,189 @@
+syntax = "proto3";
+
+package blueaccount;
+
+import "codec.proto";
+import "gogoproto/gogo.proto";
+
+// Domain represents a blue account domain.
+//
+// Domain is a container for accounts. Account representation is
+// `<name>*<domain>`.
+// Each domain owner controls all accounts that belong to that domain.
+// Each domain controls expiration of all accounts that belong to that domain.
+//
+// Because of the functionality that the Red Account provides, Blue Account
+// domain must not be `iov` as this is a domain used by Red Account
+// functionality.
+message Domain {
+  weave.Metadata metadata = 1;
+  string domain = 2;
+  // Owner is a weave.Address that controls this domain and all accounts that
+  // belong to this domain.
+  bytes owner = 3 [(gogoproto.casttype) = "github.com/iov-one/weave.Address"];
+  // Valid untill defines the expiration date for the domain. Expired domain
+  // cannot be used or modified. None of the accounts that belong to an expired
+  // domain can be used of modified as well.
+  int64 valid_untill = 4 [(gogoproto.casttype) = "github.com/iov-one/weave.UnixTime"];
```

Comment:
`valid_untill` -> `valid_until`
![you-had-one-job34-580x425](https://user-images.githubusercontent.com/11277600/67849252-575c5800-fb17-11e9-9aef-3383ecefe170.jpg)

Reply:
:sob: